include/api/Metrics.h

```diff
@@ -103,16 +125,9 @@ class Metrics
     return std::nullopt;
   }
```

Comment:
This function is not currently used.

Reply:
It's used in the tests :). But it could be used, maybe even should be used. Certainly in plugins.

Reply:
Removed.